apps/vr-tests-react-components/src/stories/DatePickerCompat.stories.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react';
+import { Steps, StoryWright } from 'storywright';
+import { DatePicker, DateRangeType } from '@fluentui/react-datepicker-compat';
+import { storiesOf } from '@storybook/react';
+import { TestWrapperDecoratorFixedWidth } from '../utilities/TestWrapperDecorator';
+
```

Comment:
You should specify the `today` prop in all vr-tests that have an open `DatePicker`, if you do not they will probably change daily breaking the PR pipeline.